src/toSafeHTMLDocument.js

```diff
@@ -12,10 +13,10 @@ export default function toSafeHTMLDocument(html: string): ?Document {
     document.implementation.createHTMLDocument
   ) {
     const doc = document.implementation.createHTMLDocument('');
-    doc.open();
-    doc.write(html);
-    doc.close();
+    if (doc.body) {
+      doc.body.innerHTML = html;
+    }
     return doc;
```

Comment:
The source HTML may contains elements that should not be placed inside `<body />`. These elements could be `<head />` `<meta />`...etc.

Alternatively, the easier solution would be just use the `DOMParser` (https://developer.mozilla.org/en-US/docs/Web/API/DOMParser) which is widely supported.

```
export default function toSafeHTMLDocument(html: string): Document {
  const parser = new window.DOMParser();
  return parser.parseFromString(html, 'text/html');
}
```